Description: Correct all mentions of the documentation path in the source code
Author: Daniele Forsi <iu5hkx@gmail.com>
Last-Update: 2021-11-03

--- a/man/aclients.1
+++ b/man/aclients.1
@@ -46,7 +46,7 @@
 
 
 .SH SEE ALSO
-More detailed information is in the pdf files in /usr/local/share/doc/direwolf, or possibly /usr/share/doc/direwolf, depending on installation location.
+More detailed information is in the PDF files at https://github.com/wb2osz/direwolf/tree/master/doc/.
 
 Applications in this package: aclients, atest, decode_aprs, direwolf, gen_packets, kissutil, ll2utm, log2gpx, text2tt, tt2text, utm2ll
 
--- a/man/atest.1
+++ b/man/atest.1
@@ -126,7 +126,7 @@
 .P
 
 .SH SEE ALSO
-More detailed information is in the pdf files in /usr/local/share/doc/direwolf, or possibly /usr/share/doc/direwolf, depending on installation location.
+More detailed information is in the PDF files at https://github.com/wb2osz/direwolf/tree/master/doc/.
 
 Applications in this package: aclients, atest, cm108, decode_aprs, direwolf, gen_packets, kissutil, ll2utm, log2gpx, text2tt, tt2text, utm2ll
 
--- a/man/decode_aprs.1
+++ b/man/decode_aprs.1
@@ -97,7 +97,7 @@
 
 
 .SH SEE ALSO
-More detailed information is in the pdf files in /usr/local/share/doc/direwolf, or possibly /usr/share/doc/direwolf, depending on installation location.
+More detailed information is in the PDF files at https://github.com/wb2osz/direwolf/tree/master/doc/.
 
 Applications in this package: aclients, atest, decode_aprs, direwolf, gen_packets, ll2utm, log2gpx, text2tt, tt2text, utm2ll
 
--- a/man/direwolf.1
+++ b/man/direwolf.1
@@ -215,7 +215,7 @@
 
 
 .SH SEE ALSO
-More detailed information is in the pdf files in /usr/local/share/doc/direwolf, or possibly /usr/share/doc/direwolf, depending on installation location.
+More detailed information is in the PDF files at https://github.com/wb2osz/direwolf/tree/master/doc/.
 
 Applications in this package: aclients, atest, cm108, decode_aprs, direwolf, gen_packets, kissutil, ll2utm, log2gpx, text2tt, tt2text, utm2ll
 
--- a/man/gen_packets.1
+++ b/man/gen_packets.1
@@ -152,7 +152,7 @@
 .P
 
 .SH SEE ALSO
-More detailed information is in the pdf files in /usr/local/share/doc/direwolf, or possibly /usr/share/doc/direwolf, depending on installation location.
+More detailed information is in the PDF files at https://github.com/wb2osz/direwolf/tree/master/doc/.
 
 Applications in this package: aclients, atest, cm108, decode_aprs, direwolf, gen_packets, kissutil, ll2utm, log2gpx, text2tt, tt2text, utm2ll
 
--- a/man/kissutil.1
+++ b/man/kissutil.1
@@ -54,7 +54,7 @@
 
 
 .SH SEE ALSO
-More detailed information is in the pdf files in /usr/local/share/doc/direwolf, or possibly /usr/share/doc/direwolf, depending on installation location.
+More detailed information is in the PDF files at https://github.com/wb2osz/direwolf/tree/master/doc/.
 
 Applications in this package: aclients, atest, decode_aprs, direwolf, gen_packets, kissutil, ll2utm, log2gpx, text2tt, tt2text, utm2ll
 
--- a/man/ll2utm.1
+++ b/man/ll2utm.1
@@ -28,7 +28,7 @@
 
 
 .SH SEE ALSO
-More detailed information is in the pdf files in /usr/local/share/doc/direwolf, or possibly /usr/share/doc/direwolf, depending on installation location.
+More detailed information is in the PDF files at https://github.com/wb2osz/direwolf/tree/master/doc/.
 
 Applications in this package: aclients, atest, decode_aprs, direwolf, gen_packets, kissutil, ll2utm, log2gpx, text2tt, tt2text, utm2ll
 
--- a/man/log2gpx.1
+++ b/man/log2gpx.1
@@ -34,7 +34,7 @@
 
 
 .SH SEE ALSO
-More detailed information is in the pdf files in /usr/local/share/doc/direwolf, or possibly /usr/share/doc/direwolf, depending on installation location.
+More detailed information is in the PDF files at https://github.com/wb2osz/direwolf/tree/master/doc/.
 
 Applications in this package: aclients, atest, decode_aprs, direwolf, gen_packets, kissutil, ll2utm, log2gpx, text2tt, tt2text, utm2ll
 
--- a/man/text2tt.1
+++ b/man/text2tt.1
@@ -58,7 +58,7 @@
 
 
 .SH SEE ALSO
-More detailed information is in the pdf files in /usr/local/share/doc/direwolf, or possibly /usr/share/doc/direwolf, depending on installation location.
+More detailed information is in the PDF files at https://github.com/wb2osz/direwolf/tree/master/doc/.
 
 Applications in this package: aclients, atest, decode_aprs, direwolf, gen_packets, kissutil, ll2utm, log2gpx, text2tt, tt2text, utm2ll
 
--- a/man/tt2text.1
+++ b/man/tt2text.1
@@ -60,7 +60,7 @@
 
 
 .SH SEE ALSO
-More detailed information is in the pdf files in /usr/local/share/doc/direwolf, or possibly /usr/share/doc/direwolf, depending on installation location.
+More detailed information is in the PDF files at https://github.com/wb2osz/direwolf/tree/master/doc/.
 
 Applications in this package: aclients, atest, decode_aprs, direwolf, gen_packets, kissutil, ll2utm, log2gpx, text2tt, tt2text, utm2ll
 
--- a/man/utm2ll.1
+++ b/man/utm2ll.1
@@ -34,7 +34,7 @@
 
 
 .SH SEE ALSO
-More detailed information is in the pdf files in /usr/local/share/doc/direwolf, or possibly /usr/share/doc/direwolf, depending on installation location.
+More detailed information is in the PDF files at https://github.com/wb2osz/direwolf/tree/master/doc/.
 
 Applications in this package: aclients, atest, decode_aprs, direwolf, gen_packets, kissutil, ll2utm, log2gpx, text2tt, tt2text, utm2ll
 
